hardhat.config.ts

```diff
@@ -62,6 +61,33 @@ const DEFAULT_CONTRACT_COMPILER_SETTINGS = {
   },
 };
 
+const networks = Object.fromEntries(
+  Object.values(CHAIN_IDs)
+    .filter((chainId) => PUBLIC_NETWORKS[chainId] !== undefined)
+    .map((chainId) => {
+      const network = PUBLIC_NETWORKS[chainId];
+      const name = network.name.replace(" ", "").toLowerCase();
+      const hubChainId = Object.values(MAINNET_CHAIN_IDs).includes(chainId) ? CHAIN_IDs.MAINNET : CHAIN_IDs.SEPOLIA;
+      const chainDef = {
+        url: network.publicRPC,
+        accounts: { mnemonic },
+        saveDeployments: true,
+        chainId: hubChainId,
+        companionNetworks: { l1: hubChainId === CHAIN_IDs.MAINNET ? "mainnet" : "sepolia" },
+      };
+
+      // zkSync is a special snowflake.
+      if ([CHAIN_IDs.ZK_SYNC, CHAIN_IDs.ZK_SYNC_SEPOLIA].includes(chainId)) {
```

Comment:
Would need to add Lens to this list. zkSync tooling is really annoying :\

Reply:
https://github.com/across-protocol/contracts/pull/833/commits/ec9f8e86f936d2f54fb69b979b65ca04bb69f56d